### PR TITLE
UN-411:Insights-to-Stories DO-NOT-MERGE

### PR DESCRIPTION
**Pull request by TNA-Allan**
Related ticket(s):
https://national-archives.atlassian.net/browse/UN-411

## About these changes
[PR-567 UN-411: Insights to Stories rename app](https://github.com/nationalarchives/ds-wagtail/pull/567) is merged into this branch
- shows Insights to Article to the Wagtail Admin User
- shows Insights to Article in code base in Wagtail areas for developer
- Added verbose names to Article attributes so that it can be updated whenever required.

## How to check these changes

As described in the ticket.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly myself before handing over to reviewer.
- [x] Included the ticket number in the PR title to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## For Reviewer

Once PR is merged, check and arrange to deploy on platform-sh.

[UN-411]: https://national-archives.atlassian.net/browse/UN-411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[UN-411]: https://national-archives.atlassian.net/browse/UN-411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by JohnHeeryTNA** [comment]
I am approving this PR from a FE and functional perspective. I have been able to run the site locally and gone through the test cases in the ticket https://national-archives.atlassian.net/browse/UN-411. I have also been able to create a new stories page.

**Review by JohnHeeryTNA — APPROVED** [review]
I am approving this PR from a FE and functional perspective. I have been able to run the site locally and gone through the test cases in the ticket https://national-archives.atlassian.net/browse/UN-411. I have also been able to create a new stories page.

File: etna/stories/blocks.py [hunk 1]
```diff
@@ -243,8 +243,8 @@ class ContentSectionBlock(SectionDepthAwareStructBlock):
 
     class Meta:
         label = "Section"
-        template = "insights/blocks/section.html"
+        template = "stories/blocks/section.html"
 
 
-class InsightsPageStreamBlock(blocks.StreamBlock):
+class StoriesPageStreamBlock(blocks.StreamBlock):
```
Comment (ababic) [inline etna/stories/blocks.py:249]:
```suggestion
class StoryPageStreamBlock(blocks.StreamBlock):
```

File: etna/stories/models.py [hunk 2]
```diff
@@ -44,52 +44,52 @@ class InsightsIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
 
     def get_context(self, request):
         context = super().get_context(request)
-        insights_pages = self.get_children().public().live().specific()
-        context["insights_pages"] = insights_pages
+        stories_pages = self.get_children().public().live().specific()
+        context["stories_pages"] = stories_pages
```
Comment (ababic) [inline etna/stories/models.py:48]:
```suggestion
        context["story_pages"] = StoryPage..objects.public().live().child_of(self)
```

File: etna/stories/models.py [hunk 3]
```diff
@@ -44,52 +44,52 @@ class InsightsIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
...
 
     class Meta:
-        verbose_name = "insights tag"
-        verbose_name_plural = "insights tags"
+        verbose_name = "stories tag"
+        verbose_name_plural = "stories tags"
 
 
-class TaggedInsights(ItemBase):
+class TaggedStories(ItemBase):
```
Comment (ababic) [inline etna/stories/models.py:71]:
```suggestion
class TaggedStory(ItemBase):
```

**Comment by ababic** [comment]
@TNA-Allan I would propose the following renames here:

`StoriesIndexPage` -> `StoryIndexPage`
`StoriesPage` -> `StoryPage`
`StoriesTag` -> `StoryTag`
`TaggedStories` -> `TaggedStory`
`stories_pages` -> `story_pages`
`stories_page ` -> `story_page`

**Comment by ababic** [comment]
@TNA-Allan I think it would be worth a conversation with Lauren to see if there's something less 'ephemeral' that we could use as a name here. `stories` feels just as likely to change as `insights`, and may not retain as much meaning as more and more types are added. Maybe we can categories these page types in a different way?

**Comment by TNA-Allan** [comment]
> @TNA-Allan I think it would be worth a conversation with Lauren to see if there's something less 'ephemeral' that we could use as a name here. `stories` feels just as likely to change as `insights`, and may not retain as much meaning as more and more types are added. Maybe we can categorize these page types in a different way?

Thanks @ababic , I have raised the rename and this in the ticket.

**Comment by jamesbiggs** [comment]
Hi @ababic, I agree, I feel like this is an issue we're having at the minute with all the new things we're making - a lot of the names are subject to user research feedback. Currently, we're working on "Record Revealed", which was called "Closer Look" until a month or two ago.
I think it's something to discuss going forwards because obviously renaming apps/pages etc is quite a large job, when it could be avoided by using a generic name for the page type. Although Lauren and the rest of the team have said that they want to have the option/ability to chop and change names of pages (e.g. insights -> stories, closer look -> record revealed) when needed.

File: config/settings/base.py [hunk 4]
```diff
@@ -383,8 +383,8 @@
 FEATURE_DOWNLOAD_RECORD_LINKS_GO_TO_DISCOVERY = strtobool(
     os.getenv("FEATURE_DOWNLOAD_RECORD_LINKS_GO_TO_DISCOVERY", "False")
 )
-FEATURE_RELATED_INSIGHTS_ON_EXPLORE_PAGES = strtobool(
-    os.getenv("FEATURE_RELATED_INSIGHTS_ON_EXPLORE_PAGES", "True")
+FEATURE_RELATED_STORIES_ON_EXPLORE_PAGES = strtobool(
+    os.getenv("FEATURE_RELATED_STORIES_ON_EXPLORE_PAGES", "True")
```
[the remark this answers is not in the document]
Comment (TNA-Allan) [reply config/settings/base.py]: This is now changed to reflect categorised name `Article`, am aware thanks, will make it explicit in ticket.

File: etna/stories/blocks.py (hunk 1 above)
Comment (TNA-Allan) [reply etna/stories/blocks.py]: This is now changed to reflect categorised name `Article`

File: etna/stories/models.py [hunk 5]
```diff
@@ -20,18 +20,18 @@
 
 from ..heroes.models import HeroImageMixin
 from ..teasers.models import TeaserImageMixin
-from .blocks import FeaturedCollectionBlock, InsightsPageStreamBlock
+from .blocks import FeaturedCollectionBlock, StoriesPageStreamBlock
 
 
-class InsightsIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
-    """InsightsIndexPage
+class StoriesIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
```
[the remark this answers is not in the document]
Comment (TNA-Allan) [reply etna/stories/models.py]: This is now changed to reflect categorised name `Article`

File: etna/stories/models.py [hunk 6]
```diff
@@ -44,52 +44,52 @@ class InsightsIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
...
     content_object = ParentalKey(
-        to="insights.InsightsPage",
+        to="stories.StoriesPage",
         on_delete=models.CASCADE,
         related_name="tagged_items",
     )
 
 
-class InsightsPage(
+class StoriesPage(
```
[the remark this answers is not in the document]
Comment (TNA-Allan) [reply etna/stories/models.py]: This is now changed to reflect categorised name `Article`

File: etna/stories/models.py (hunk 2 above)
Comment (TNA-Allan) [reply etna/stories/models.py]: This is now changed to reflect categorised name `Article`.  Also ideally is not specific to this ticket in terms of variable assignment. Have condensed the assignment but retained the original expression.
`context["article_pages"] = self.get_children().public().live().specific()`

File: etna/stories/models.py (hunk 3 above)
Comment (TNA-Allan) [reply etna/stories/models.py]: This is now changed to reflect categorised name `Article`

**Comment by TNA-Allan** [comment]
> > @TNA-Allan I think it would be worth a conversation with Lauren to see if there's something less 'ephemeral' that we could use as a name here. `stories` feels just as likely to change as `insights`, and may not retain as much meaning as more and more types are added. Maybe we can categorize these page types in a different way?
> 
> Thanks @ababic , I have raised the rename and this in the ticket.

`Insights` and `Stories` now categorised as `Article` more details in the ticket.

**Comment by TNA-Allan** [comment]
Closed - follow on[ PR-601](https://github.com/nationalarchives/ds-wagtail/pull/601)

**Pull request closed without merging**